script/download_MNIST.py

```diff
@@ -63,4 +93,4 @@
         'test_image': test_image,
         'test_label': test_label,}
 with open("./data/MNIST.pkl", 'wb') as file_handle:
-    pickle.dump(data, file_handle)
+    pickle.dump(data, file_handle)
```

Comment:
```suggestion
    pickle.dump(data, file_handle)

```

Adding new line character at end of file to fix the red symbol